data/json/items/magazine/40mm.json

```diff
@@ -0,0 +1,11 @@
+[
+  {
+    "id": "belt40mm",
```

Comment:
40mm ammo looks quite bulky - those values are probably too low. Could an average person even wear that many?